parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -302,8 +283,6 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     KUBELET_REGISTER_SCHEDULABLE={{WrapAsVariable "registerSchedulable"}}
 {{end}}
 
-MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
```

Comment:
This implementation vector has been officially retired in favor of the standard `{{CloudInitData "<file reference>"}}` usage pattern. See `/etc/systemd/system/kubelet.service` and `/etc/systemd/system/kms.service` above.